Delivered-To: mine260309@gmail.com
Received: by 2002:a4f:7f0c:0:0:0:0:0 with SMTP id a12csp61772ivd;
        Thu, 3 Jan 2019 13:58:43 -0800 (PST)
X-Google-Smtp-Source: ALg8bN7WM+1zVBxTVGTUZRqyOXp8nmk0s/bW052kzqKMvApKg4DjqjRR8vsN/Lh24y4UHwklN8jF
X-Received: by 2002:aa7:8758:: with SMTP id g24mr48970611pfo.250.1546552723828;
        Thu, 03 Jan 2019 13:58:43 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1546552723; cv=none;
        d=google.com; s=arc-20160816;
        b=d+/ihmVLfIliWNsxs8OwbxETN0/gvScnwuwcHmntdU/NbhuWsaLC8YmKJ/vWpbJGm6
         C3V7gGqk6LndmWPXBEWfYiwJ1Sjy9Ajm5C4dfpyCTsyNq1jOWf5XqwU7dt9Y7LDjL+8z
         BGr4e7p9yHnB6k1vHFQq8G6kn1AGhubqULqepTvR2fFozgNzAxBGyTayu+jGxWOcSv+f
         8q/oH/omj1VQ+OvT5GyTjkElJlm+uYwt8kvri+c4kKa2Lo2tUca5eJdeJ8XUQIuwVPBF
         o5c8Q83tSDGD+mcUw/Lee/+U7rMLfC8ATQOAJrIdS7G+WgCObfQ6kiwfNtk+ALweFJ6H
         0y0Q==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=sender:errors-to:cc:list-subscribe:list-help:list-post:list-archive
         :list-unsubscribe:list-id:precedence:message-id:references
         :in-reply-to:date:subject:to:from:delivered-to;
        bh=T2McmzzWEcVhoUR/GmA1q8IhYkWMdnWxdEcAdyda2Xw=;
        b=v01A792GFUb44fxjqP5kA1GRKJPnBDJbb2BO3ETlxUnU4Nn/Dm/0D58hcYipj+edkL
         fUbJxT1xwmjccB9V/32eFkMgOxfs711r0Eesa6wqF0ECL9E5IHGWhAHJ0O1ArpFyh3KO
         9hcNkYo67O1qZ3d4H6vaxs4PvvUcCCN1MMfvba+CZsQMwidc4LkBUh/wqLQ/ehKqgTNm
         bLw5tvTDbaGNA27c0e68AJI33vbJ7UNP9Ngvlvx3mf/OuNq7SWKx54QkdwgNvQ1vQcVr
         zoyQ75ZQhnLbrz8obLzLkirX44A8F61lCjRaQzTdVEOFOyeW4DVXnu9mtPlLLGhfRoCE
         ms1w==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Return-Path: <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>
Received: from lists.ozlabs.org (lists.ozlabs.org. [203.11.71.2])
        by mx.google.com with ESMTPS id i4si13859857pfg.218.2019.01.03.13.58.43
        for <mine260309@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-CHACHA20-POLY1305 bits=256/256);
        Thu, 03 Jan 2019 13:58:43 -0800 (PST)
Received-SPF: pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) client-ip=203.11.71.2;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Received: from lists.ozlabs.org (lists.ozlabs.org [IPv6:2401:3900:2:1::3])
	by lists.ozlabs.org (Postfix) with ESMTP id 43W1yT3VxBzDqLl
	for <mine260309@gmail.com>; Fri,  4 Jan 2019 08:58:41 +1100 (AEDT)
X-Original-To: openbmc@lists.ozlabs.org
Delivered-To: openbmc@lists.ozlabs.org
Authentication-Results: lists.ozlabs.org;
 spf=pass (mailfrom) smtp.mailfrom=linux.ibm.com
 (client-ip=148.163.156.1; helo=mx0a-001b2d01.pphosted.com;
 envelope-from=eajames@linux.ibm.com; receiver=<UNKNOWN>)
Authentication-Results: lists.ozlabs.org;
 dmarc=none (p=none dis=none) header.from=linux.ibm.com
Received: from mx0a-001b2d01.pphosted.com (mx0a-001b2d01.pphosted.com
 [148.163.156.1])
 (using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
 (No client certificate requested)
 by lists.ozlabs.org (Postfix) with ESMTPS id 43W1ss2t7XzDqKw
 for <openbmc@lists.ozlabs.org>; Fri,  4 Jan 2019 08:54:41 +1100 (AEDT)
Received: from pps.filterd (m0098393.ppops.net [127.0.0.1])
 by mx0a-001b2d01.pphosted.com (8.16.0.22/8.16.0.22) with SMTP id
 x03LsZIa001684
 for <openbmc@lists.ozlabs.org>; Thu, 3 Jan 2019 16:54:39 -0500
Received: from e35.co.us.ibm.com (e35.co.us.ibm.com [32.97.110.153])
 by mx0a-001b2d01.pphosted.com with ESMTP id 2psn3sybxp-1
 (version=TLSv1.2 cipher=AES256-GCM-SHA384 bits=256 verify=NOT)
 for <openbmc@lists.ozlabs.org>; Thu, 03 Jan 2019 16:54:38 -0500
Received: from localhost
 by e35.co.us.ibm.com with IBM ESMTP SMTP Gateway: Authorized Use Only!
 Violators will be prosecuted
 for <openbmc@lists.ozlabs.org> from <eajames@linux.ibm.com>;
 Thu, 3 Jan 2019 21:54:21 -0000
Received: from b03cxnp07029.gho.boulder.ibm.com (9.17.130.16)
 by e35.co.us.ibm.com (192.168.1.135) with IBM ESMTP SMTP Gateway: Authorized
 Use Only! Violators will be prosecuted; 
 (version=TLSv1/SSLv3 cipher=AES256-GCM-SHA384 bits=256/256)
 Thu, 3 Jan 2019 21:54:19 -0000
Received: from b03ledav003.gho.boulder.ibm.com
 (b03ledav003.gho.boulder.ibm.com [9.17.130.234])
 by b03cxnp07029.gho.boulder.ibm.com (8.14.9/8.14.9/NCO v10.0) with ESMTP id
 x03LsIdM15728836
 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-GCM-SHA384 bits=256 verify=FAIL);
 Thu, 3 Jan 2019 21:54:18 GMT
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 551C46A04F;
 Thu,  3 Jan 2019 21:54:18 +0000 (GMT)
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 10E6A6A057;
 Thu,  3 Jan 2019 21:54:18 +0000 (GMT)
Received: from talon7.ibm.com (unknown [9.41.179.222])
 by b03ledav003.gho.boulder.ibm.com (Postfix) with ESMTP;
 Thu,  3 Jan 2019 21:54:17 +0000 (GMT)
From: Eddie James <eajames@linux.ibm.com>
To: openbmc@lists.ozlabs.org
Subject: [PATCH dev-4.19 4/6] ARM: dts: aspeed: witherspoon: Enable video
 engine
Date: Thu,  3 Jan 2019 15:54:06 -0600
X-Mailer: git-send-email 1.8.3.1
In-Reply-To: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
References: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
X-TM-AS-GCONF: 00
x-cbid: 19010321-0012-0000-0000-000016F368BB
X-IBM-SpamModules-Scores: 
X-IBM-SpamModules-Versions: BY=3.00010339; HX=3.00000242; KW=3.00000007;
 PH=3.00000004; SC=3.00000272; SDB=6.01141310; UDB=6.00593988; IPR=6.00921536; 
 MB=3.00024957; MTD=3.00000008; XFM=3.00000015; UTC=2019-01-03 21:54:20
X-IBM-AV-DETECTION: SAVI=unused REMOTE=unused XFE=unused
x-cbparentid: 19010321-0013-0000-0000-000055ADA67B
Message-Id: <1546552448-22621-5-git-send-email-eajames@linux.ibm.com>
X-Proofpoint-Virus-Version: vendor=fsecure engine=2.50.10434:, ,
 definitions=2019-01-03_10:, , signatures=0
X-Proofpoint-Spam-Details: rule=outbound_notspam policy=outbound score=0
 priorityscore=1501
 malwarescore=0 suspectscore=1 phishscore=0 bulkscore=0 spamscore=0
 clxscore=1015 lowpriorityscore=0 mlxscore=0 impostorscore=0
 mlxlogscore=895 adultscore=0 classifier=spam adjust=0 reason=mlx
 scancount=1 engine=8.0.1-1810050000 definitions=main-1901030187
X-BeenThere: openbmc@lists.ozlabs.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: Development list for OpenBMC <openbmc.lists.ozlabs.org>
List-Unsubscribe: <https://lists.ozlabs.org/options/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=unsubscribe>
List-Archive: <http://lists.ozlabs.org/pipermail/openbmc/>
List-Post: <mailto:openbmc@lists.ozlabs.org>
List-Help: <mailto:openbmc-request@lists.ozlabs.org?subject=help>
List-Subscribe: <https://lists.ozlabs.org/listinfo/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=subscribe>
Cc: Eddie James <eajames@linux.vnet.ibm.com>
Errors-To: openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org
Sender: "openbmc" <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>

From: Eddie James <eajames@linux.vnet.ibm.com>

Enable the video engine and set it's required memory-region property

Signed-off-by: Eddie James <eajames@linux.vnet.ibm.com>
---
 arch/arm/boot/dts/aspeed-bmc-opp-witherspoon.dts | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-opp-witherspoon.dts b/arch/arm/boot/dts/aspeed-bmc-opp-witherspoon.dts
index b933ced..2ed09da 100644
--- a/arch/arm/boot/dts/aspeed-bmc-opp-witherspoon.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-opp-witherspoon.dts
@@ -28,7 +28,7 @@
 		};
 
 		gfx_memory: framebuffer {
-			size = <0x01000000>;
+			size = <0x04000000>;
 			alignment = <0x01000000>;
 			compatible = "shared-dma-pool";
 			reusable;
@@ -634,6 +634,15 @@
 	memory-region = <&gfx_memory>;
 };
 
+&video {
+	status = "okay";
+	memory-region = <&gfx_memory>;
+};
+
+&vhub {
+	status = "okay";
+};
+
 &pinctrl {
 	aspeed,external-nodes = <&gfx &lhc>;
 };
-- 
1.8.3.1

